Bigtable/src/Table.php

```diff
@@ -0,0 +1,352 @@
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function exists(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Fetch a fresh representation of the table from the service.
+     *
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.GetTableRequest GetTableRequest
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#table Table
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function reload(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Return the table state.
+     *
+     * When tables are created or updated, they may take some time before
+     * they are ready for use. This method allows for checking whether an
+     * table is ready.
+     *
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function state(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Create a table.
+     *
+     * Example:
+     * ```
+     * $tableInfo = $table->create();
+     * ```
+     *
+     * @todo add options for ColumnFamily
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#google.bigtable.admin.v2.CreateTableRequest CreateTableRequest
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array $options [optional] Configuration options.
+     *
+     * @return array Table information
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function create(array $options = [])
+    {
+        $instanceName = TableAdminClient::instanceName($this->projectId, $this->instanceId);
+        $this->info = $this->connection->createTable([
+            'parent' => $instanceName,
+            'tableId' => $this->id
+        ] + $options);
+
+        return $this->info;
+    }
+
+    /**
+     * Delete table.
+     *
+     * Example:
+     * ```
+     * $table->delete();
+     * ```
+     *
+     * @param array $options [optional] Configuration options.
+     *
+     * @return void
+     */
+    public function delete(array $options = [])
+    {
+        return $this->connection->deleteTable([
+            'name' => $this->name
+        ] + $options);
+    }
+
+    /**
+     * Add columnFamily.
+     *
+     * Example:
+     * ```
+     * $tableInfo = $table->addColumnFamilys(['cf1'=>[],'cf2'=>[],'cf3'=>[]]);
+     * ```
+     * @todo add GCRule options
+     *
+     * @param array[] $columnFamilys List of ColumnFamily, key is name of column family,
+     *        value is optional GCRule
```

Comment:
Can you please remove "value is optional GCRule" since it's not supported yet?

Reply:
done